website/notifications/events/model.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
```

Comment:
By convention, `model.py` constains MODM models. Maybe rename this module to `base.py`?